### DISCUSSION TITLE
Add `latest` tag for Docker images

### DISCUSSION THREAD

**Discussion by nwithan8** [LizardByte/community#544] [Sunshine]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the latest beta/pre-release?

This issue is present in the latest pre-release

### Describe the Bug

No `latest` and/or consistent tag on Docker Hub pointing to a latest/master/development image.

### Expected Behavior

A `latest` tag that always points to the latest stable Docker image for Sunshine. `latest` is commonly used, but does not need to be explicitly this.

### Host Operating System

Docker

### Operating System Version

N/A

### Architecture

other, n/a

### Sunshine commit or version

N/A

### Package

Linux - Docker

### GPU Type

n/a

### GPU Model

N/A

### GPU Driver/Mesa Version

N/A

### Capture Method

None

### Relevant log output

```shell
N/A
```

**Comment by ReenigneArcher** [comment]
Sure we do, but it depends on which distro and distro version.

latest: https://hub.docker.com/r/lizardbyte/sunshine/tags?name=latest-
master: https://hub.docker.com/r/lizardbyte/sunshine/tags?name=master-

Edit: if you're specifically referring to v0.23.1 (for example), yea we don't have a tag that points to that... and at this point I don't really have a good way to do that since a beta/stable are technically created the same exact way. The only difference is the GitHub pre-release is edited to become a stable release.

**Reply by ReenigneArcher** [reply]
Also converted this to a feature request.

**Reply by nwithan8** [reply]
How does the distro of the host machine affect the Docker container to the point that there are different Docker images depending on the host OS?

EDIT: To be transparent, I'm trying to build a Docker template to easy-install Sunshine on Unraid, which is based on Slackware, so I'm not entirely sure which image variant would work best.

**Reply by ReenigneArcher** [reply]
I don't believe the host machine distro should affect the one selected for Sunshine necessarily. It's just a matter of preference with what software you want to install along side Sunshine and what desktop you'd want to use when streaming Sunshine.

It might not be clear but these images are not meant to stream the host's desktop or anything like that, but intended to be more like a virtual machine. Although if the hosts were the same, I guess you could volume map the sunshine binary... I don't think that would work for Unraid if that's your intent.

These might also be of interest to you:

- https://github.com/Steam-Headless/docker-steam-headless (uses Sunshine)
- https://github.com/linuxserver/docker-steamos (uses Sunshine)
- https://github.com/games-on-whales/wolf (alternative to Sunshine)

**Comment by ReenigneArcher** [comment]
Regarding the request for stable tags, I suppose it could be possible depending on if it's possible to edit existing published images/tags.

These are some of the workflows we use to change states when we convert a beta/pre-release to stable.

- ReadTheDocs: https://github.com/LizardByte/Sunshine/blob/master/.github/workflows/update-docs.yml
- Flathub sync: https://github.com/LizardByte/Sunshine/blob/master/.github/workflows/update-flathub-repo.yml
- Homebrew: https://github.com/LizardByte/Sunshine/blob/master/.github/workflows/update-homebrew-release.yml
- Pacman repo: https://github.com/LizardByte/Sunshine/blob/master/.github/workflows/update-pacman-repo.yml
- Winget: https://github.com/LizardByte/Sunshine/blob/master/.github/workflows/update-winget-release.yml

Perhaps something similar could work for Docker, although I think it would be a bit more complicated. We should probably also cleanup old beta tags (we only keep the latest 2 in GitHub)
